src/atoms/footer/footer.tsx

```diff
@@ -22,23 +24,26 @@ const PoweredBy = styled.a`
   font-size: ${(props) => props.theme.font.size.xxs};
 `;
 
-const StyledFooter = styled("footer")<FooterTypes>`
+const StyledFooter = styled("footer") <FooterTypes>`
   ${(props) => genStyles(baseStyles, props.style, props)}
 `;
 
-const BaseFooter = (props: FooterTypes & { theme?: DefaultTheme }) => {
-  const { text, children, style } = props;
+const BaseFooter = ({ text, children, style, config, theme }: FooterTypes) => {
```

Comment:
add types. If `style` isn't being used, remove it

Reply:
style is being used, do you mean theme?